### PR TITLE
Supercharge copy and UI changes

### LINKED ISSUE #1170: Supercharge UI changes
### What would you like?
Make UI changes to the supercharge screen.

Figma mockups: https://www.figma.com/file/31HUGhIHLXEm1qwVpHV1QR/CELO-Rewards?node-id=2169%3A51088

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Made UI changes to the supercharge screen.

Figma mockups: https://www.figma.com/file/31HUGhIHLXEm1qwVpHV1QR/CELO-Rewards?node-id=2169%3A51088

Screenshot of these changes:
Screen Shot 2021-09-22 at 17 45 26

### Other changes

N/A

### Tested

Manually using phone numbers from EU and non-EU countries.

### How others should test

Go to the rewards screen and make sure you see the UI changes.
If you have an european phone number you should see cEUR as a currency and cUSD otherwise.

### Related issues

- Fixes #1170

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/locales/en-US/consumerIncentives.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   "saveMoreEarnMore": {
-    "header": "Cash in to earn more",
-    "text": "Deposit up to {{rewardsMax}} cUSD or {{rewardsMax}} cEUR and earn {{percent}}% anually."
+    "header": "Boost your balance",
+    "text": "Deposit {{maxBalance}} {{currency}} to max-out your {{percent}}% annual rewards."
   },
-  "conclusion": "It’s that simple. Start earning today.",
-  "addCusd": "Add cUSD",
+  "conclusion": "It’s that simple. Supercharge your crypto today!",
+  "addCusd": "Add {{currency}}",
```
Comment (lisaqyli) [inline packages/mobile/locales/en-US/consumerIncentives.json:18]: minor comment: might want to update `addCUSD` to `addCurrency` to better reflect the prop now

File: packages/mobile/src/app/saga.ts [hunk 2]
```diff
@@ -156,6 +156,7 @@ export interface RemoteFeatureFlags {
   rewardsPercent: number
   rewardsStartDate: number
   rewardsMax: number
+  rewardsMin: number
```
Comment (tarikbellamine) [inline packages/mobile/src/app/saga.ts:159]: How have we been handling migrations/typing for these new remote config state variables? Technically, without a migration this type should be `undefined | number` right? In practice I understand that most users will connect to firebase on app load and have this populated.

Comment (gnardini) [reply packages/mobile/src/app/saga.ts:159]: On the app reducer we have this:
```
case REHYDRATE: {
      // Ignore some persisted properties
      const rehydratePayload = getRehydratePayload(action, 'app')
      return {
        ...state,
        ...rehydratePayload,
        appState: initialState.appState,
        locked: rehydratePayload.requirePinOnAppOpen ?? initialState.locked,
        sessionId: '',
      }
    }
```
`state` in this case is the initial state, which has the default value for `rewardsMin` of `10` so I don't think this value will ever be `undefined` when read.

**Review by tarikbellamine — APPROVED** [review]

**Comment by ValoraQA** [comment]
Hey @MuckT , We have verified this issue using Latest Android Internal build V1.20.2  & updated the status in Test Quality Tool.
**Test Quality Tool Link:**
**Android(9.0):**
Case 1: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516289
Case 2: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516290

**New Query:**
- We have observed that user is able to see "cUSD" in description of Reward page even user logged in with European Country phone number. We have raised #1206 mkl

**Verified On Devices:** Redmi Note 8(9.0)

CC: @gnardini

**Comment by ValoraQA** [comment]
Hey @MuckT , We have verified this task using Latest Android Internal build V1.20.2 & updated the status in Test Quality Tool.
Test Quality Tool Link:
**Android(11.0):**
Case 1: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516257
Case 2: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516258

**Android(8.0):**
Case 1: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516305
Case 2: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516306 

**New Query:**

- We have observed that user is able to see "cUSD" in description of Reward page even user logged in with European Country phone number. We have raised #1206 for same

**Verified On Devices:** OnePlus 7T(11.0), Samsung Galaxy J5(8.0)

CC: @gnardini

**Comment by ValoraQA** [comment]
Hey @MuckT We have verified this issue using Test flight build V1.120.2 & updated status in Test Quality Tool 

**Test Quality Tool:**
- https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516321
**Verified On Devices:** iPhone Xs Max(13.5.1)

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this task using Test flight build V1.120.2 & updated status in Test Quality Tool

**Test Quality Tool:**
**iOS(15.0):**
Case1: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516353
Case2: https://valora.testquality.com/project/12128/plan/21930/run/150772/result/2516354
**Devices:** iPhone 12 (15.0)
CC: @gnardini

**Pull request merged**